stl/src/winapinls.cpp

```diff
@@ -5,10 +5,495 @@
 
 #if _STL_WIN32_WINNT < _WIN32_WINNT_VISTA
 
-#include "nlsdownlevel.h"
 #include <ctype.h>
 #include <stdlib.h>
 
+namespace {
+struct LCIDTOLOCALENAME {

```

Comment:
Did you clang-format this? I'm surprised that clang-format didn't indent the contents of the unnamed namespace. (Anyway, if the CI run complains about clang-format, you'll know why ;))